modules/services/images/src/main/res/drawable/ic_headphone.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="22dp"
+    android:height="19dp"
```

Comment:
Can we export this as 24dp by 24dp to match the other icons on the settings page?

Reply:
Updated in d1b9cf9